src/parser/jobs/blm/modules/RotationWatchdog.js

```diff
@@ -210,6 +204,20 @@ export default class RotationWatchdog extends Module {
 				</Trans>,
 			}))
 		}
+		// Suggestion to not use manafont before Despair
+		if (this._mfBeforeDespair) {
+			this.suggestions.add(new TieredSuggestion({
+				icon: ACTIONS.MANAFONT.icon,
+				content: <Trans id="blm.rotation-watchdog.suggestions.mf-before-despair.content">
+					Using <ActionLink {...ACTIONS.MANAFONT} /> before <ActionLink {...ACTIONS.DESPAIR} /> leads to less <ActionLink {...ACTIONS.DESPAIR} />s than possible being cast. Try to avoid that since <ActionLink {...ACTIONS.DESPAIR} /> is stronger than <ActionLink {...ACTIONS.FIRE_IV} />.
```

Comment:
Suuuuuper minor nitpick, but it's more grammatically correct to use 'fewer' instead of 'less' since it's a countable thing we're talking about. Other than that, it's good.

Reply:
me not good england